test/blueprints/Do.Test.Blueprints.Service.Test/DataAccess/PersistingEntity.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Do.Test.DataAccess;
+
+public class PersistingEntity : TestServiceSpec
+{
+    [Test]
+    public void Created_entity_persists()
+    {
+        var entity = GiveMe.AnEntity();
+
+        entity.ShouldBeInserted();
+    }
+
+    [Test]
+    public void Entity_does_not_persist_when_deleted()
```

Comment:
improve case name, its not "not persisting", we are deleting an already persisted entity, so its about deleting, not "not persisting"